src/application/F3DStarter.cxx

```diff
@@ -108,8 +107,6 @@ int F3DStarter::Start(int argc, char** argv)
         this->LoadFile(f3d::loader::LoadFileEnum::LOAD_LAST);
         return true;
       });
-    this->Internals->Engine->getWindow().setWindowName(f3d::AppTitle);
-    this->Internals->Engine->getWindow().setIcon(F3DIcon, sizeof(F3DIcon));
```

Comment:
whats wrong with doing that here ?

Reply:
It's in the application, and we don't have access to `AppTitle` and `F3DIcon` symbols.

Reply:
F3DIcon is pure applicative side. AppTitle should be put into the application at some point. It is not libf3d responsibility to do that. This is why I added the api to f3d::window

Reply:
see f3d_config.h